code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -433,6 +434,10 @@
 			S.switch_mode_to(TRACK_INFILTRATOR)
 		countdown.start()
 		set_security_level(SEC_LEVEL_DELTA)
+
+		if (proper_bomb) // Why does this exist
+			countdown_music = play_soundtrack_music('sound/soundtrack/countdown.ogg', specific_z = SSmapping.levels_by_trait(ZTRAIT_STATION)[1])
```

Comment:
playing this at max volume could get a bit irritating after a while. It shouldn't be any higher than 60 IMO

Reply:
I'll test out some volume levels tomorrow

Reply:
Title music plays at 85, I think down to 80 is a good level especially since it isn't immediately loud